webapp/app/js/config/sample.local.conf.js

```diff
@@ -3,17 +3,19 @@
  * file, you can obtain one at http://mozilla.org/MPL/2.0/. */
 
 'use strict';
+var production = "https://treeherder.mozilla.org";
+var stage = "https://treeherder.allizom.org";
+var dev = "http://treeherder-dev.allizom.org";
+var same_host = "";
```

Comment:
Hm, if it's always going to be vagrant, I wonder about `localvagrant`. Maybe that communicates both things. It just seemed the use of 'same' implied it was the same as something else (that is not known).